### ISSUE TITLE
Unable to open the curriculum pdf documents

### ISSUE THREAD

**Issue by mvaskuri** [cncf/curriculum#69]
There is an error popping up while open the pdf documents. 
[CKA_Curriculum_v1.27.pdf](https://github.com/cncf/curriculum/files/11714578/CKA_Curriculum_v1.27.pdf)

**Comment by za** [comment]
The issue seems valid https://github.com/cncf/curriculum/blob/master/CKA_Curriculum_v1.27.pdf I can't open it too.

<img width="663" alt="Screenshot 2023-08-29 at 16 52 30" src="https://github.com/cncf/curriculum/assets/409455/c9da6705-2766-4132-9773-8ae7eef01d9b">

**Comment by aungzanbaw** [comment]
It's fine (CKA,CKAD,CKS) on my Chrome browser(Mac). Simply reload the browser, try with incognito mode or disable some plugins.

**Comment by za** [comment]
Ah, this time is working fine. I can open it now @aungzanbaw 

Maybe, if it's possible to provide the source code instead of the PDF file. #suggestion #noturgent

![Screenshot 2023-08-30 121849](https://github.com/cncf/curriculum/assets/409455/a4b4614a-ebf6-4c84-9262-9457c6d1e3fd)

**Comment by aungzanbaw** [comment]
@za indeed, [.md] file would be easily digestable for everyone and editable.

**Comment by za** [comment]
Do you know the source of the PDF document @aungzanbaw ? Maybe we can initiate a markdown file.

**Comment by aungzanbaw** [comment]
This [commit](https://github.com/cncf/curriculum/commit/acb43587b95a95dd9aeb9b0ca5be0ae0b6ac39ed) push directly into the repo, .md won't be rich as compare to .pdf but that's depends on the owners.

**Comment by duoarc** [comment]
@zuzannapn This Curriculum issue seems to be recurring and might warrant a deeper investigation. At the time of posting this comment, I have not been able to view CKS curriculum for about 4 hours now. I have tried getting the rawgithubcontent link. I tried checking older commits, nothing works.

It can be quite frustrating because I need to review the curriculum for any gaps in my knowledge today before the craziness of the week kicks in. 

I am willing to assist in probably converting the Curriculum to markdown so we can have markdown version available. Or look into the source of the PDF files and figure out why they don't load sometimes.

**Comment by zuzannapn** [comment]
@duoarc the latest version of CKS curriculum is here: [https://github.com/cncf/curriculum/blob/master/CKS_Curriculum_%20v1.29.pdf](url)

**Comment by ChristopherHanson** [comment]
If you're having trouble with the PDF, you can always go to the CKS page at the Linux Foundation: https://training.linuxfoundation.org/certification/certified-kubernetes-security-specialist/

What is on that page is the canonical version of the domains & competencies seen in the PDF.
